internal/cri/util/util.go

```diff
@@ -29,17 +29,21 @@ import (
 	crilabels "github.com/containerd/containerd/v2/internal/cri/labels"
 	clabels "github.com/containerd/containerd/v2/pkg/labels"
 	"github.com/containerd/containerd/v2/pkg/namespaces"
+	"github.com/containerd/containerd/v2/pkg/timeout"
 	"github.com/containerd/log"
 )
 
-// deferCleanupTimeout is the default timeout for containerd cleanup operations
-// in defer.
-const deferCleanupTimeout = 1 * time.Minute
+// deferCleanupTimeout is the timeout for containerd cleanup operations in defer.
+const deferCleanupTimeout = "io.containerd.timeout.cri.defercleanup"
```

Comment:
```suggestion
const deferCleanupTimeoutKey = "io.containerd.timeout.cri.defercleanup"
```

Reply:
// deferCleanupTimeoutKey is used to retrieve the configurable timeout value for containerd cleanup operations in defer.

Reply:
done.thanks